pkg/package/toml.go

```diff
@@ -85,7 +85,17 @@ func (dep *Dependency) MarshalTOML() string {
 	source := dep.Source.MarshalTOML()
 	var sb strings.Builder
 	if len(source) != 0 {
-		sb.WriteString(fmt.Sprintf(DEP_PATTERN, dep.Name, source))
+		sourceWithoutBrace := strings.TrimSuffix(source, "}")
```

Comment:
I think that the logic of serializing kcl.mod to toml is incorrect, maybe you should add some test cases to make sure this is correct. Because from a functional point of view, you are only adding a field, why are you adding "{}" again here ?

Reply:
if we directly pass it as a field it the `kpm.mod` will look like,
```
[dependencies]
catalog = { git = "https://github.com/KusionStack/catalog.git", tag=“0.0.3”}, version = “0.1.1” 
```

as the other fields are comin in from git source and if we try to append the version in the git source then the `kpm.mod.lock` will have two version field, as done in this PR(#406 )

Reply:
Hi @Gmin2 😃

I see what you mean, but what I mean is that if you want to change the process of generating "{}" to the upper level, you should remove the original generating "{}", but I can't see your removal of the generating "{}" in the diff. I suggest that you write some unit tests for your functional changes first, so that I can intuitively judge whether your changes meet the expectations according to the unit tests, and then review your specific implementation. Just write the test for `MarshallTOML()`.